packages/features/CalendarEventBuilder.ts

```diff
@@ -0,0 +1,269 @@
+import type { Prisma } from "@prisma/client";
+import type { TFunction } from "next-i18next";
+
+import type { TimeFormat } from "@calcom/lib/timeFormat";
+import type { SchedulingType } from "@calcom/prisma/enums";
+import type { CalendarEvent, Person, CalEventResponses, AppsStatus } from "@calcom/types/Calendar";
+import type { VideoCallData } from "@calcom/types/VideoApiAdapter";
+
+export class CalendarEventBuilder {
```

Comment:
Calendar event builder class would handle all the updation and creation of the properties